src/api/operator/types.ts

```diff
@@ -80,19 +65,7 @@ export type Order = Pick<RawOrder, 'owner' | 'uid' | 'appData' | 'kind' | 'parti
 /**
  * Raw API trade response type
  */
-export type RawTrade = {
-  blockNumber: number
-  logIndex: number
-  owner: string
-  txHash: string
-  orderUid: string
-  buyAmount: string
-  sellAmount: string
-  sellAmountBeforeFees: string
-  buyToken: string
-  sellToken: string
-  executionTime: string
-}
+export type RawTrade = TradeMetaData | (TradeMetaData & { executionTime: string })
```

Comment:
Similar to the above, except that `executionTime` is not returned by the API yet.
Was a request but it has not been fulfilled yet AFAICT, so it can be commented out

Otherwise, if it is included, then it should be added to the SDK